wl/utils/Utils.java

```diff
@@ -47,19 +47,38 @@ public static int bash(String... args) throws Exception {
 		for (String a : args) System.out.print(" " + a);
 		System.out.println();
 
-		Process p = Runtime.getRuntime().exec(args);
-		p.waitFor();
+		ProcessBuilder pb = new ProcessBuilder(args);
+		pb.redirectErrorStream​(true);
+		Process p = pb.start();
+
 		BufferedReader b = new BufferedReader(new InputStreamReader(p.getInputStream()));
 		String str;
-		while ((str = b.readLine()) != null) {
-			System.out.println("    # " + str);
-		}
+		while ((str = b.readLine()) != null) System.out.println("    # " + str);
```

Comment:
Is this autoformat, or did you remove the { } ?

Reply:
I removed it because it looks cleaner like this IMHO